flamingpy/decoders/unionfind/uf_classes.py

```diff
@@ -141,13 +141,14 @@ def grow(self, edge):
             self.status[frozenset(edge)] = "half-grown"
         elif edge_status == "half-grown":
             self.status[frozenset(edge)] = "grown"
-            return edge
+
+        return edge
```

Comment:
Need to double check with @PriNad that this does not cause any undesirable behaviour.

Reply:
Correct!

Reply:
Safe scenario is to return `None` because currently this is what happens. Anyhow, best to ask and make things feel consistent.

Reply:
@sduquemesa, what you have suggested last is the right way of doing this and is 100% safe -- this will not change Pryia's function. I'll implement it.